checks.d/kong.py

```diff
@@ -0,0 +1,70 @@
+# stdlib
+import urlparse
+
+# 3rd party
+import requests
+import simplejson as json
+
+# project
+from checks import AgentCheck
+from util import headers
+
+
+class Kong(AgentCheck):
+    """ collects metrics for Kong """
+    def check(self, instance):
+        metrics = self._fetch_data(instance)
+        for row in metrics:
+            try:
+                name, value, tags = row
+                self.gauge(name, value, tags)
+            except Exception:
+                self.log.error(u'Could not submit metric: %s' % repr(row))
```

Comment:
Could you do something like this instead : ` self.log.error(u'Could not submit metric: %s', row)` ? Because of https://www.simonmweber.com/2014/11/24/python-logging-traps.html